plugin/trino-mongodb/src/main/java/io/trino/plugin/mongodb/MongoSessionProperties.java

```diff
@@ -39,6 +40,11 @@ public MongoSessionProperties(MongoClientConfig mongoConfig)
                         "Read only required fields from a row type",
                         mongoConfig.isProjectionPushdownEnabled(),
                         false))
+                .add(booleanProperty(
```

Comment:
why we want this as session property, looks like in other connectors, expressions enabled by default?

Reply:
Currently, there are a couple of correctness issues with the current implementation. So for now, Made the complex pushdown disabled by dfault. After some time, we can enable it by default if we don't encounter any correctness issue.